docs/hub/webhooks.md

```diff
@@ -0,0 +1,120 @@
+
+You can configure the webhooks in your [settings](https://huggingface.co/settings/webhooks).
+
+You can watch repo changes or community events. If you watch all repos belonging to a user or an organization, you can also receive events for repo creation.
+
+## Webhook payloads
+
+After registering a webhook, you will be notified of events via an `HTTP POST` call on the specified URL. The payload is encoded in JSON.
+
+To determine the nature of the event, you can look at the `event` and `domain` fields in the payload.
+
+You can check the last payloads sent in the activity tab of the webhook page, as well as replay webhooks.
+
+### Event & Domain
+
+The top-level properties `event` and `domain` are always specified and used to determine the nature of the event.
+
+`event` can take one of the following values: 
+
+- `"create"`
+- `"delete"`
+- `"update"`
+- `"move"`
+
+`domain` specifies the domain of the event. It can be one of the following values:
+
+
+- `["repo"]` - Global events on repos: create, delete, move
+- `["repo", "content"]` - Events on the content of the repo, like new commits or tags. Triggers on new pull requests as well due to the newly created reference / commit.
+- `["repo", "config"]` - Events on the config: update space secrets, update settings, update DOIs, disabled or not, ...
+- `["community", "discussion"]` - Creating a discussion or pull request, updating the title or status
+- `["community", "discussion", "comment"]` - Creating, updating, hiding a comment
+
+⚠️ **You should consider that any event on a more granular domain is an `"update"` on the encompassing domains.** ⚠️
+
+For example, any event on `["community", "discussion", "..."]` is an `"update"` on `["community", "discussion"]`.
+
+This is important in case more granularity is added to the domains by Hugging Face. For example if the domain domain `["repo", "config", "doi"]` is added, it would be accompanied by an `"add"` event when DOIs are created. 
+
+If you follow the rule above, your application will treat any event on `["repo", "config", "doi"]` as an `"update"` on `["repo", "config"]`, and will not break if Hugging Face adds more granularity to its domains.
+
+### Repo
+
+In the current version of webhooks, the top level property `repo` is always specified, as events can always be associated to a repo. Sample payload:
+
+```json
+"repo": {
+	"type": "model",
+	"name": "some-user/some-repo",
+	"gitalyUid": "583c43bd0009f32d976f788446d5010ed6ea4af0c6d0235358378cb56d312f2b",
+	"id": "6366c000a2abcdf2fd69a080",
+	"private": false,
+	"url": {
+		"web": "http://huggingface.co/some-user/some-repo",
+		"api": "http://huggingface.co/api/models/some-user/some-repo"
+	},
+	"headSha": "c379e821c9c95d613899e8c4343e4bfee2b0c600",
+	"tags": [
+		"license:other",
+		"has_space"
+	],
+	"author": {
+		"id": "61d2000c3c2083e1c08af22d"
+	}
+}
+```
+
+`headSha` is only sent when the domain starts with `"repo"`, it's not sent on community events.
```

Comment:
Sorry what are community events? Any event on a domain that includes "community"?

Reply:
it's events on discussions and PRs, ie. in the "Community" tab

maybe that "community events" is not very clear, and Discussions and PRs is more explicit. cc @gary149 too